### ISSUE TITLE
Dark mode: Header text in "how to fix" has insufficient contrast against its background on mouse hover

### ISSUE THREAD

**Issue by iamrafan** [microsoft/accessibility-insights-windows#845]
**Describe the bug**
The header text in "how to fix" section in the right pane has insufficient contrast against its
background on mouse hover

**To Reproduce**
1. Make sure color setting for default app mode in windows is set to 'Dark'
2. Launch AI-Windows
3. Scan any app and navigate to "Results in UI Automation Tree"
4. Hover the mouse on the headers "Rule" and "Issues" under the test details section in "how to fix" tab
5. Observe that the header text has insufficient contrast against its hovered background

**Expected behavior**
The header text should have contrast ratio of at least 4.5:1 against its background

**Actual behavior**
The contrast ratio for the header text is less than 4.5:1

**Screenshots or .GIF**
![5](https://user-images.githubusercontent.com/22068434/88111245-6d58ef80-cb62-11ea-8cf7-87085a05abdc.png)

**Desktop**
 - OS: Windows 10 Enterprise, Version 1903
 - Accessibility Insights for Windows Version: Production Release: v 1.1.1248.1

**Comment by ferBonnin** [comment]
reactivating to allow original issue creator to close. 
@iamrafan could you please regress and close?

**Comment by DaveTryon** [comment]
@iamrafan, this is in any build after https://github.com/microsoft/accessibility-insights-windows/releases/tag/v1.1.1780.01
